pineappl_capi/src/lib.rs

```diff
@@ -1867,3 +1895,77 @@ pub unsafe extern "C" fn pineappl_grid_convolve(
         mu_scales,
     ));
 }
+
+/// Get the number of convolutions for a given Grid.
+///
+/// # Safety
+///
+/// If `grid` does not point to a valid `Grid` object, for example when `grid` is the null pointer,
+/// this function is not safe to call.
+#[no_mangle]
+pub unsafe extern "C" fn pineappl_convolutions_len(grid: *mut Grid) -> usize {
```

Comment:
This should be `pineappl_grid_convolutions_len`.

Reply:
Still takes a while to fully get used to the naming convention 🙈, mainly because I was not fully consistent before in some places.